cmd/prometheus/config.go

```diff
@@ -190,45 +187,6 @@ func init() {
 		"Local storage engine. Supported values are: 'persisted' (full local storage with on-disk persistence) and 'none' (no local storage).",
 	)
 
-	// Remote storage.
-	cfg.fs.StringVar(
```

Comment:
From the "What's not included" section in https://docs.google.com/document/d/1TFhL6kEasDxkk3pxsHcM6TqSx4E4NI3X2kshNNQ6-X0/edit:

"Current remote storage code for Influx/OpenTSDB/Graphite"